scripts/ci/pre_commit/pre_commit_insert_extras.py

```diff
@@ -81,7 +81,7 @@ def process_documentation_files() -> bool:
     extra_type_dict = get_extra_types_dict()
     for file, file_format, add_comment in FILES_TO_UPDATE:
         if not file.exists():
-            raise Exception(f"File {file} does not exist")
+            raise FileNotFoundError(f"File {file} does not exist")
```

Comment:
This looks fine. @uranusjr ?

Reply:
I’d even consider not raising this at all since it’d fail when you try to read the file anyway, but this is fine.